src/ci_providers/provider_buildkite.js

```diff
@@ -0,0 +1,126 @@
+// https://buildkite.com/docs/guides/environment-variables
```

Comment:
can we remove this comment ish?

Reply:
Yeah sure
```suggestion
```

Reply:
Sorry, I meant all of it for all the methods as in the other providers that aren't the template

Reply:
I actually like them, so as when they change, we know where to look to see what they changed to. 🤷‍♀️